client/src/app/models/InventoryItem.ts

```diff
@@ -0,0 +1,17 @@
+export interface InventoryItem {
+  _id: number;
+  Location: string;
+  Plant: string;
+  Zone: string;
+  Aisle: string;
+  Part_Number: string;
+  Part_Description: string;
+  Serial_Number: string;
+  Condition: string;
+  Category: string;
+  Owner: string;
+  Criticality: string;
+  Average_Cost: string;
+  Quantity: string;
+  Unit_of_Measure: string;
+}
```

Comment:
Again, I think this is something that we need to generate dynamically

Reply:
I don't think we need to generate an interface anymore for it in the new implementation.